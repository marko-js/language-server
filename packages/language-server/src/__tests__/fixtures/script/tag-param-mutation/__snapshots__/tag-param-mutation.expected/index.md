## Hovers
### Ln 1, Col 7
```marko
> 1 | <let/string="bar"/>
    |       ^ const string: string
  2 |   //  ^?
  3 | <test-tag|val|>
  4 |   <button onClick() { string = val }>
```

### Ln 4, Col 24
```marko
  2 |   //  ^?
  3 | <test-tag|val|>
> 4 |   <button onClick() { string = val }>
    |                        ^ (property) string: string
  5 |                   //   ^?      ^?
  6 |     ${val}
  7 |   </button>
```

### Ln 4, Col 32
```marko
  2 |   //  ^?
  3 | <test-tag|val|>
> 4 |   <button onClick() { string = val }>
    |                                ^ (parameter) val: string
  5 |                   //   ^?      ^?
  6 |     ${val}
  7 |   </button>
```

