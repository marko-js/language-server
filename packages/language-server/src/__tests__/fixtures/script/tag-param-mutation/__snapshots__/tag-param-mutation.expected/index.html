<div>
  <button data-marko-node-id="2" onClick>
                  
    placeholder
  </button>
</div>